thoth/storages/data/alembic/versions/419dadacc3f7_added_entry_in_.py

```diff
@@ -0,0 +1,36 @@
+"""Added entry in KebechetGithubInstallation table for environment name.
+
+Revision ID: 419dadacc3f7
+Revises: 95c40432360f
+Create Date: 2021-07-21 19:38:18.803976+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "419dadacc3f7"
+down_revision = "95c40432360f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column("has_symbol", "software_environment_id", existing_type=sa.INTEGER(), nullable=False)
+    op.alter_column("has_symbol", "versioned_symbol_id", existing_type=sa.INTEGER(), nullable=False)
+    op.add_column("kebechet_github_installations", sa.Column("runtime_environment_name", sa.Text(), nullable=True))
+    op.create_unique_constraint(None, "kebechet_github_installations", ["slug", "runtime_environment_name"])
+    op.create_unique_constraint(None, "python_package_version_entity_rule", ["id"])
```

Comment:
Some of these do not look relevant to this change. Please remove them. We should fix this issue as it has been here for some time.